### DISCUSSION TITLE
Collection Level Admin

### DISCUSSION THREAD

**Discussion by hardikns** [outline/outline#4655] [Self Hosting] [answered]
It would nice to have a admin for each collections instead of global admins. This would allow different leaders/moderators of a collection manage access to those collections.

**Comment by ghost** [comment]
agree

**Comment by tommoor** [comment, marked as answer]
There are plans for this on the roadmap. Can't give any firm dates but it's a desirable feature

**Comment by hardikns** [comment]
@tommoor - is this possible as an addon or something?

**Reply by tommoor** [reply]
I'm not sure what you mean. The functionality hasn't been built yet :)

**Reply by ghost** [reply]
Hi Tom, that we understand. But it's becoming a difficult to manage admin access. We need an ability for providing admin access at collection group? We have multiple teams, and giving admins to many to manage access request at collection level makes it quite difficult and does not help in our Security management policies.

**Reply by ghost** [reply]
So if we can have a quick 'add-in' feature developed that could be used, it will be great. I see that the access level already supports Wiki level and collection level. For instance we noticed if someone is invited to Wiki by 'Read' only access, even if we provide 'View and Edit' access to a collection level, 'Read' only access at Wiki level overrides the collection level access..so I guess the code / framework already there, it's a question of extending the 'admin' feature at Wiki and collection level..
